docs/userman/gui/extension/dynamic_element/list_of_value_props.md

```diff
@@ -0,0 +1,154 @@
+In the previous section, we discussed creating dynamic elements. In Taipy GUI, lists can be bound to Python variables or
+expressions, allowing real-time updates in the UI. Custom elements that handle lists should define their properties using
+the `PropertyType^` class to specify the format of list values. This setup supports multidimensional data binding, ensuring
+that any changes to the list automatically refresh the display.
+
+# Using list of values
+
+In this section, we will expand the dynamic element library by adding a new dynamic custom element.
```

Comment:
I would certainly give a rough idea of what we plan to do, before we explain how.

Like : "we will build a visual element that can represent a list of blahblah..." see?